docs/tutorials/super-for-jq.md

```diff
@@ -449,7 +470,7 @@ But more powerfully, types can be used anywhere a value can be used and
 in particular, they can be group-by keys, e.g.,
 ```mdtest-command
 echo '{x:1,y:2}{s:"foo"}{x:3,y:4}' |
-  super -f table -c "count() by \`shape\`:=typeof(this) | sort count" -
+  super -f table -c "count() by this['shape']:=typeof(this) | sort count" -
```

Comment:
I felt like showing backticks while having to escape them to prevent interference by the shell was a little esoteric, so here I opted to instead show the field reference with `this['...']` which still feels a little esoteric for a tutorial doc but a little less so IMO. I'm open to debate though. 😄